src/cardJson.js

```diff
@@ -0,0 +1,26 @@
+db.cards.insertMany([
```

Comment:
Do we need this file?

Reply:
Yes we can leave it like this for now. Its going to be used to provide sample data. It is also a backup in case we want to remove everything.